libraries/botbuilder-ai/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "lib": ["es2015"],
+    "lib": ["es2015", "dom"],
```

Comment:
We don't want to add `"dom"` to our tsconfig.json as this can force changes on downstream TypeScript consumers of our libraries. See Issue #1565 and PR #1575